spec.html

```diff
@@ -25626,7 +25626,7 @@ <h1>The Error Constructor</h1>
         <li>is <dfn>%Error%</dfn>.</li>
         <li>is the initial value of the *"Error"* property of the global object.</li>
         <li>creates and initializes a new Error object when called as a function rather than as a constructor. Thus the function call `Error(&hellip;)` is equivalent to the object creation expression `new Error(&hellip;)` with the same arguments.</li>
-        <li>is designed to be subclassable. It may be used as the value of an `extends` clause of a class definition. Subclass constructors that intend to inherit the specified Error behaviour must include a `super` call to the Error constructor to create and initialize subclass instances with an [[ErrorData]] internal slot.</li>
+        <li>may be used as the value of an `extends` clause of a class definition. Subclass constructors that intend to inherit the specified Error behaviour must include a `super` call to the Error constructor to create and initialize subclass instances with an [[ErrorData]] internal slot.</li>
```

Comment:
Actually, I do see and use subclassing on the Error constructor. Isn't it _is_ designed subclassable?

Reply:
Error doesn't have any methods outside of `toString`, which side steps the issues with subclassing. Still I don't see any reason to deviate from the proposed language here. I see this PR as removing encouragement, and IMO, the fact that Error can be successfully subclassed doesn't mean the spec needs to encourage it over other built-ins that can't be easily successfully subclassed.

Further, by removing the encouragement, we're also saying something about future proposals for extending Error: that they shouldn't accommodate subclassing beyond the constructor being usable in `extends`.